ports/duckdb/portfile.cmake

```
@@ -0,0 +1,73 @@
+set(DUCKDB_SHORT_HASH 401c8061c6)
+vcpkg_from_github(
+        OUT_SOURCE_PATH SOURCE_PATH
+        REPO duckdb/duckdb
+        REF v${VERSION}
+        SHA512 ecd036ff975e90c4e9cc3a25784169f5938db19eacd2abc201719d329ec1055608bb2270de22bf409d828196ede6ffe95369154c916916e13a7a14071b05652e
+        HEAD_REF master
+)
+
+# Remove vendored dependencies which are not properly namespaced
+file(REMOVE_RECURSE
+    "${SOURCE_PATH}/third_party/catch"
+    "${SOURCE_PATH}/third_party/imdb"
+    "${SOURCE_PATH}/third_party/snowball"
+    "${SOURCE_PATH}/third_party/tpce-tool"
+)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" DUCKDB_BUILD_STATIC)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" DUCKDB_BUILD_DYNAMIC)
+
+set(EXTENSION_LIST "autocomplete;httpfs;icu;json;parquet;tpcds;tpch")
+set(EXTENSION_LOAD_LIST "")
```

```suggestion
set(BUILD_EXTENSIONS "")
```

and rename elsewhere to match https://github.com/duckdb/duckdb/blob/f317a112e0c789a11987d9de6d621d8469c00fb8/CMakeLists.txt#L1255C22-L1255C38 ?